order/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+class OrderItem(models.Model):
+    product_id = models.ForeignKey('product.Product', on_delete=models.CASCADE)
+    order_id = models.ForeignKey('Order', on_delete=models.CASCADE)
+    order_item_quantity = models.IntegerField()
```

Comment:
이미 order item table에 있는 속성이므로 `quantity`
```suggestion
    quantity = models.IntegerField()
```

Reply:
네네!
혹시 밑에 order_item_price 변수도 price로 변경하면 될까요?

Reply:
네 민영님!